app/assets/stylesheets/govuk_publishing_components/components/_intervention.scss

```diff
@@ -21,3 +21,32 @@
   height: 1em;
   margin-bottom: -2px;
 }
+
+// stylelint-disable declaration-no-important
+@include govuk-media-query($media-type: print) {
+  .gem-c-intervention {
+    break-inside: avoid;
+    background-color: transparent;
+    border-color: $govuk-print-text-colour;
+
+    .govuk-body:has(+ .govuk-body .js-dismiss-link) {
```

Comment:
`.govuk-body:has(+ .govuk-body .js-dismiss-link)` seems quite complicated. Now that we're hiding the dismiss link, I think there'll only be one `.govuk-body` element shown in print, so could this be simplified? e.g.

```CSS
    .govuk-body {
      margin: 0;
    }
```

Reply:
I did consider that, but if there's a possibility of more than one paragraph with `.govuk-body` then `margin: 0` will remove all of the spacing between paragraphs which we wouldn't want.

Reply:
There isn't a possibility of more than one paragraph, see https://github.com/alphagov/govuk_publishing_components/blob/main/app/views/govuk_publishing_components/components/_intervention.html.erb

While I was looking at that file, I noticed something else - the `govuk-!-display-none-print` class is on the `govuk-link` element, but it should be on the wrapping `govuk-body` element (at the moment it'll not show the link in print, but will show the 'empty' wrapping paragraph, which could include margin).

Reply:
OK, thanks, yes your solution is better, that's now updated.